deployment/docker/victorialogs/journald/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM ubuntu:24.04
+FROM debian:forky
 
 RUN \
   apt update && \
   apt install -y \
     systemd \
     systemd-journal-remote && \
-  sed -i 's/# URL=/URL=http:\/\/victorialogs:9428\/insert\/journald/g' /etc/systemd/journal-upload.conf && \
+  sed -i 's/# URL=/URL=http:\/\/victorialogs:9428\/insert\/journald\nHeader=AccountID: 10\nHeader=ProjectID: 10/g' /etc/systemd/journal-upload.conf && \
```

Comment:
You don't have to change it, but using `echo '<>' > /file` would be a cleaner, for example:

```sh
echo '[Upload]
URL=http://victorialogs:9428/insert
Header=AccountID: 10
Header=ProjectID: 10
' > /etc/systemd/journal-upload.conf
```